partner_external_map/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,23 @@
+In the old days of Odoo/OpenERP, back in version 6.1, there was an official
```

Comment:
Is this text relevant for version 16 ? I think few people played with version 6.1..

Reply:
@legalsylvain maybe you can suggest what the text should be then?

Reply:
No idéa. I dont use it. Just describe what the module does ?

Reply:
I'm suggesting a replacement below.